src/cameraGeom/Camera.cc

```diff
@@ -31,6 +31,35 @@ namespace cameraGeom {
 
 namespace {
 
+/**
+ * Get a transform from one TransformMap
+ *
+ * `fromSys` and `toSys` must both be present in the same TransformMap, but that TransformMap may be from
+ *    any detector or this camera object.
+ *
+ * @param[in] fromSys  Camera coordinate system of input points
```

Comment:
Missing description for `camera`.

Reply:
This is actually just the revert part of the unrelated change you asked me to put on another commit, and this function will be removed entirely on what is now a separate commit.